### ISSUE TITLE
Unclear documentation for isolation

### ISSUE THREAD

**Issue by bdcaf** [stoically/temporary-containers#392]
I'm having a hard time figuring out the settings for *Navigation - Target Domain* and *Navigation - Mouse*.

Maybe it's because I'm not a native speaker so I don't pick up the grammar. But I don't believe there is a problem labeling them with simpler grammar.

For *Navigation - Target Domain* the options *Different from Tab Domain* and *Different from Tab Domain & Subdomain* appear to do the opposite what I would expect. Not sure why the & is not used in the logical sense here - in the meaning "isolate unless domain AND subdomain match".

For *Mouse* all the settings default to *Never* - what I think is a bad default for left click.  Apparently despite having *Never* selected I get isolation. Not  sure what changes when I alter this setting.

**Comment by stoically** [comment]
I'm not exactly happy with the documentation either - happy to accept PRs / suggestions!

> Not sure why the & is not used in the logical sense here - in the meaning "isolate unless domain AND subdomain match".

That's what the preference should be doing. As long as "Domain & its Subdomain" are a "Navigation Target" it should not isolate. If that isn't the case I'd need steps to reproduce.

> For Mouse all the settings default to Never - what I think is a bad default for left click. Apparently despite having Never selected I get isolation. Not sure what changes when I alter this setting.

Did you check the ?-bubble-tooltips and/or the [mouse click](https://github.com/stoically/temporary-containers/wiki/Global-Isolation#mouse-click). "Never" in this case is meant to say, that those mouse clicks are never isolated (by default).

**Comment by bdcaf** [comment]
I'm fairly certain it is working as expected but here's explicitly the behavior I am seeing:

As example on https://neocities.org/browse (in temporary tab 125) if I click on https://fauux.neocities.org/. 
When I set the option for Navigation to "Different from Tab Domain & Subdomains" the (left) click opens the link in the same tab 125.
When the option is set to "Different from Tab Domain" the link opens a new temporary tab (127).  Confusing for me as I would understand both pages sharing the same domain.

About the mouse I experience following. Currently on a notebook so I can only try "left click" and "Ctrl-Left". If I set it to "Always" I get every link in a new tab - as expected. If I set it to "Never" I still get a new tabs according to the setting in Navigation.  I have Navigation set to "Different from Tab Domain & Subdomains" and left click to "Different from Tab Domain" I get the effect of "Different from Tab Domain" - i.e. open in a new tab. 
If I switch around and set Navigation to "Different from Tab Domain" and left mouse to "Different from Tab Domain & Subdomains" I also get the "Different from Tab Domain" effect.

It is more clear as you describe here. From the wiki alone I had the impression the Mouse setting would override the one for Navigation.

**Comment by stoically** [comment]
> Confusing for me as I would understand both pages sharing the same domain but not subdomain.

Yeah, can see that. The preference is meant to be read in combination with the preference header "Target Domain". So: "Target Domain Different from Tab Domain".

> From the wiki alone I had the impression the Mouse setting would override the one for Navigation.

Yeah, the opposite is true. Navigation always overrides mouse clicks.

> It could be something like:

The problem here indeed is the question of how people perceive the short preference name. I guess at the end of the day you can't name the short version of the preference in way that describes it in a way that all people perceive it correct - and it rather needs an documentation clearly describing the effects instead. I'll try to address the confusions in the tooltips and in the wiki.
